Felienne/hedy
# [UI] Adds Hedy Certificate of Completion

**Description**

This PR adds the Hedy Certificate of completion. For that a few changes have been done:

* Adds a new achievement: the Hedy certificate
* Adds a new page accessed through `/certificate/<username>`
* Changes the public profile page to add a crown if the user has completed Hedy.
* Changes the landing page to add a crown if the user has completed Hedy.
* Adds the lines of a program to the things we track about programs to display the longest program a user has done.
* Changes the ending of the quiz in the last level to tell the user they have completed Hedy.

Hedy Certificate Page
![image](https://user-images.githubusercontent.com/45865185/196054393-18a9a5f9-01d8-407b-8185-0818d26018a9.png)
 Crown in the Landing Page 
![image](https://user-images.githubusercontent.com/45865185/196054475-ba467743-a982-4c48-bcc5-36478c9b3fcc.png)
Crown and button to access certificate in public profile
![image](https://user-images.githubusercontent.com/45865185/196054509-f9f30386-34fa-4ef9-9771-91fd29271cf3.png)
Change in last quiz:
![image](https://user-images.githubusercontent.com/45865185/196054692-9cf63a93-6f22-4da5-a2a2-7e249417331d.png)


**Fixes _issue or discussion number_**

Fixes #3360

**How to test**

1. You need to add a quiz for level 18
2. Once the quiz you can access your account certificate.
3. Go to your public profile and see the new button.
4. Go to your landing page and see your crown!

**Checklist**
Done? Check if you have it all in place using this list:*
  

If you're unsure about any of these, don't hesitate to ask. We're here to help!
